Builds/VisualStudio2017/README.md

```diff
@@ -242,19 +219,35 @@ cmake ..\.. -G"Visual Studio 15 2017 Win64" -DBOOST_ROOT="C:\lib\boost_1_68_0" -
 Now launch Visual Studio 2017 and select **File | Open | Project/Solution**.
 Navigate to the `build\cmake` folder created above and select the `rippled.sln`
 file. You can then choose whether to build the `Debug` or `Release` solution
-configuration. Within the **Solution Explorer**, selected either the `rippled`
-(unity build) project or the `rippled_classic` (non-unity) project, and
-right-click to build.
+configuration.
 
 The executable will be in 
 ```
-.\build\cmake\Release\rippled(_classic).exe
+.\build\cmake\Release\rippled.exe
+```
+or
+```
+.\build\cmake\Debug\rippled.exe
 ```
- or
-````
-.\build\cmake\Debug\rippled(_classic).exe
-````
-where these paths are relative to your cloned git repository.
+These paths are relative to your cloned git repository.
+
+# Unity/No-Unity Builds
+
+The rippled build system defaults to using [unity source files](http://onqtam.com/programming/2018-07-07-unity-builds/)
+to improve build times. In some cases it might be desirable to disable the unity build and compile
+individual translation units. Here is how you can switch to a "no-unity" build configuration:
+
+## Visual Studio Integrated CMake
+ 
+Edit your `CmakeSettings.json` (described above) by adding `-Dunity=OFF` to the `cmakeCommandArgs` entry 
+for each build configuration.
+
+## Standalone CMake Builds
+
+When running cmake to generate the Visual Studio project files, add `-Dunity=OFF` to the 
+command line options passed to cmake.
+
+**Note:** you will need to re-run the cmake configuration step anytime you want to switch between unity/no-unity builds.
```

Comment:
Do you need to clear the CMake cache too, or is it okay?

Reply:
FWIW, I usually build my unity and non-unity CMake projects in separate directories, so it's not an issue. But I did the experiment generating with `-unity=ON`, building, `-unity=OFF`, and building again. Both builds worked.